source/includes/fact-download-queries.rst

```diff
@@ -0,0 +1,54 @@
+Allows you to download completed conversions in the 
+:guilabel:`Query Converter` pane.
+
+Individual Query
+~~~~~~~~~~~~~~~~
+
+.. procedure::
+   :style: normal
+
+   .. step:: Select your query
+
+      a. From the :guilabel:`Code Generation` tab, click the 
+         :guilabel:`Query Converter` pane.
+
+         .. note::
+
+           If you are not already logged into your Atlas account or have
+           an expired session, you must login to proceed.
+
+      #. On the left-hand :guilabel:`Queries` pane, select a query, view,
+         or stored procedure.
+
+      #. If your query has not been converted, select a 
+         :guilabel:`Target Language` and click :guilabel:`Convert`
+         :icon-lg:`Sparkle`.
+
+   .. step:: Download the converted query
+
+      a. Click the :icon-fa5:`download` icon to download the selected 
+         query conversion.
```

Comment:
(minimalism)
Click :icon-fa5:`download` to download the ...
(can omit "the" and "icon" if you'd like.

Globally. You can often omit "the" and button or toggle or whatever to keep the sentence a smidge simpler.